torch/_dynamo/side_effects.py

```diff
@@ -514,6 +510,16 @@ def codegen_update_mutated(self, cg: PyCodegen):
                         cg(value)
                         cg(var.mutable_local.source)
                         suffixes.append([create_instruction("STORE_ATTR", argval=name)])
+            elif isinstance(var, variables.TupleIteratorVariable):
+                for _ in range(var.index):
+                    cg.extend_output([create_load_global("next", push_null=True)])
```

Comment:
This assumes the user doesn't have a variable named `next`, which is unsound.

Reply:
I'll add a special dynamo util for this